site/docs/education/egeria-dojo/running-egeria/user-interfaces/ecosystem-ui/ecosystem-ui-start.md

```diff
@@ -2,8 +2,6 @@
 
 With the simple environment we have set up, we can now start exploring the Egeria Ecosystem UI.
 
-In later labs you can refer back to this documentation and try out the new with the new environment.
```

Comment:
Did you find this sentence unhelpful and intended to delete?
Or was it an accidental deletion? I can see there are typos in it to correct.

Reply:
Yeah I asked David for his opinion on how to reword the sentence and he recommended to just remove the sentence as it didn't make much sense